Scenario Based Samples/Iframe Full Server Embedding/EmbededShell/IframeFullServerSample/Views/_ViewImports.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿@using IframeFullServerSample
```

Comment:
Check all the Views are needed for this sample? If not, please remove.

Reply:
Checked and removed which are not needed